### DISCUSSION TITLE
Reels of multiple units in purchase order confusion

### DISCUSSION THREAD

**Discussion by loki42** [inventree/InvenTree#6711] [Q&A]
If I buy 40000 capacitors, on reels of 4000, I am not sure how to receive them / specify them in the PO so that it creates 10 stock items when I receive the PO instead of 1 of 40000. If I specify a supplier pack size (which is maybe how I'm meant to do it) then I have to specify that I bought 10 reels, not that I bought 40000 parts.  Is there a way to specify the number of individual parts purchased and then have it split to a pack quantity when received? Maybe I'm just trying the wrong thing in the wrong area.

**Comment by SchrodingersGat** [comment]
I would suggest that either:

a) you add 10x separate line items - one for each reel
b) you "receive" the item 10x at the lower quantity - this will create 10x individual received items
